### PR TITLE
Adding more supported spatial operations 

### PR DESCRIPTION
**Pull request by oshawa-connection**
As discussed is issue #59. This is my first ever PR, and I'm not familiar with the internals of the library but here is my attempt.
E.g. I'm not sure if I should list "geom.AsBinary()" as I'm not sure if this gets translated to ST_AsBinary.

### CODE REVIEW

File: conceptual/EFCore.PG/mapping/nts.md [hunk 1]
```diff
@@ -88,37 +88,52 @@ Note that the plugin is far from covering all spatial operations. If an operatio
 .NET                             | SQL
 ---------------------------------|-----
 geom.Area()                      | [ST_Area(geom)](https://postgis.net/docs/manual-3.0/ST_Area.html)
+geom.AsBinary()                  | [ST_AsBinary(geom)](https://postgis.net/docs/manual-3.0/ST_AsBinary.html)
 geom.AsText()                    | [ST_AsText(geom)](https://postgis.net/docs/manual-3.0/ST_AsText.html)
 geom.Boundary                    | [ST_Boundary(geom)](https://postgis.net/docs/manual-3.0/ST_Boundary.html)
+geom1.Centroid(geom2))           | [ST_Centroid(geom1, geom2)](https://postgis.net/docs/manual-3.0/ST_Centroid.html)
```
Comment (roji) [inline conceptual/EFCore.PG/mapping/nts.md:94]: I think Centroid is a property, not a method accepting another geometry. ]On the PostGIS side](https://postgis.net/docs/ST_Centroid.html) this is just a function accepting a single geomtry.

File: conceptual/EFCore.PG/mapping/nts.md [hunk 2]
```diff
@@ -88,37 +88,52 @@ Note that the plugin is far from covering all spatial operations. If an operatio
...
 geom1.Contains(geom2))           | [ST_Contains(geom1, geom2)](https://postgis.net/docs/manual-3.0/ST_Contains.html)
+geomCollection.Count             | [ST_NumGeometries(geom1)](https://postgis.net/docs/manual-3.0/ST_NumGeometries.html)
```
Comment (roji) [inline conceptual/EFCore.PG/mapping/nts.md:96]: We also translate Count on line strings (to ST_NumPoints)

**Review by roji — CHANGES_REQUESTED** [review]
Thanks, that looks pretty good! See minor comments below.

>  E.g. I'm not sure if I should list "geom.AsBinary()" as I'm not sure if this gets translated to ST_AsBinary.

Yeah, this happens [here](https://github.com/npgsql/efcore.pg/blob/main/src/EFCore.PG.NTS/Query/ExpressionTranslators/Internal/NpgsqlNetTopologySuiteMethodCallTranslatorPlugin.cs#L119).

File: conceptual/EFCore.PG/mapping/nts.md (hunk 1 above)
Comment (oshawa-connection) [reply conceptual/EFCore.PG/mapping/nts.md]: Yes you're right. Fixed in the latest commit.

File: conceptual/EFCore.PG/mapping/nts.md (hunk 2 above)
Comment (oshawa-connection) [reply conceptual/EFCore.PG/mapping/nts.md:97]: Also added in the latest commit, as well as the operations found in [this file](https://github.com/npgsql/efcore.pg/blob/main/src/EFCore.PG.NTS/Query/ExpressionTranslators/Internal/NpgsqlNetTopologySuiteMethodCallTranslatorPlugin.cs).

**Review by roji — APPROVED** [review]
Thanks for this! It's great to have our docs more up-to-date here!

**Pull request merged**